Contributing.md

```diff
@@ -18,6 +18,18 @@ As an organization contributor, you can merge other people's pull requests, or o
 
 This project is not continuously deployed, there is space for debate after review too. Offering everyone the chance to revert, or make an amending pull request. If it feels right, merge.
 
+#### Bug fixes and Release cycles
+
+We maintain two major branches: `master` and `develop`.
```

Comment:
"major" doesn't _quite_ capture what I think you're trying to say here. I would say "persistent" or "permanent" maybe? We might want to add that they are protected as well.

> We maintain two permanent, protected branches: `master` and `develop`.